core/src/filed/fd_plugins.h

```diff
@@ -61,6 +61,13 @@ struct BareosFilePacket;
 struct FindFilesPacket;
 #endif /* FILE_DAEMON */
 
+enum IoStatus : int
+{
+  error = -1,
+  success = 0,
+  do_io_in_core = 1,
+};
```

Comment:
doesn't that expose `error` and `success` to the global scope?

Reply:
Good point, solved by putting into own namespace.